pages/_app.js

```diff
@@ -1,16 +1,24 @@
-import { nanoid } from "nanoid";
 import Head from "next/head";
-import { useState } from "react";
+import { useEffect, useState } from "react";
 import GlobalStyles from "../components/GlobalStyles";
-import { initialEvents } from "../data/sampleEvents";
 
 function MyApp({ Component, pageProps }) {
-  // dummy data state
-  const [sampleEvents, setSampleEvents] = useState(initialEvents);
+  // store data
+  const [sampleEvents, setSampleEvents] = useState();
 
   // show create page
   const [isCreating, setIsCreating] = useState(false);
 
+  // fetch data from database
```

Comment:
This works out just fine. Just as an idea (maybe for later refactoring): You could move the fetch into an own component and just call it from here (as your own custom hook)

Reply:
That´s very helpful thanks! I will put it in the same story like component testing 👌🏽